mitosheet/mitosheet/api/get_imported_files_and_dataframes_from_current_steps.py

```diff
@@ -50,7 +52,8 @@ def get_import_data_with_single_import_list(step_type: str, params: Dict[str, An
                 'file_name': params['file_name'],
                 'sheet_names': [sheet_name],
                 'has_headers': params['has_headers'],
-                'skiprows': params['skiprows']
+                'skiprows': params['skiprows'],
+                'decimal': params['decimal']
```

Comment:
I am guessing that you forgot to add these when you initially did this PR. I am wondering if there is some way to test we are making changes here effectively -- even if it's just adding a comment somewhere!

Reply:
Check out these tests, it catches this! https://github.com/mito-ds/monorepo/pull/509/commits/da3d93c74b98fc4e8a12e8070732a6d9b0a28767

You're right! I did forget at first so I added this test.